packages/contracts-bedrock/contracts/test/CrossDomainOwnable3.t.sol

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Bytes32AddressLib } from "@rari-capital/solmate/src/utils/Bytes32AddressLib.sol";
+import { CommonTest, Messenger_Initializer } from "./CommonTest.t.sol";
+import { CrossDomainOwnable3 } from "../L2/CrossDomainOwnable3.sol";
+import { AddressAliasHelper } from "../vendor/AddressAliasHelper.sol";
+import { Hashing } from "../libraries/Hashing.sol";
+import { Encoding } from "../libraries/Encoding.sol";
+
+contract XDomainSetter3 is CrossDomainOwnable3 {
+    uint256 public value;
+
+    function set(uint256 _value) external onlyOwner {
+        value = _value;
+    }
+}
+
+contract CrossDomainOwnable3_Test is Messenger_Initializer {
+    XDomainSetter3 setter;
+
+    function setUp() public override {
+        super.setUp();
+        vm.prank(alice);
+        setter = new XDomainSetter3();
+    }
+
```

Comment:
needs a test saying alice is the owner using the owner getter

Reply:
added an `assertEq` in `setUp`